tests/integration/test_actions.py

```diff
@@ -0,0 +1,123 @@
+import os
+import pytest
+from pytest_operator.plugin import OpsTest
+import logging
+
+log = logging.getLogger(__name__)
+
+certs = [
+    "ETCDCTL_KEY_FILE=/var/snap/etcd/common/client.key",
+    "ETCDCTL_CERT_FILE=/var/snap/etcd/common/client.crt",
+    "ETCDCTL_CA_FILE=/var/snap/etcd/common/ca.crt",
+    "ETCDCTL_KEY=/var/snap/etcd/common/client.key",
+    "ETCDCTL_CERT=/var/snap/etcd/common/client.crt",
+    "ETCDCTL_CACERT=/var/snap/etcd/common/ca.crt",
+]
+
+
+@pytest.mark.abort_on_fail
+async def test_build_and_deploy(ops_test: OpsTest):
+    charm = await ops_test.build_charm(".")
+    await ops_test.model.deploy("cs:~containers/easyrsa-441")
```

Comment:
lets use a modern charm version:

```suggestion
    await ops_test.model.deploy("easyrsa")
```